tracee-ebpf/tracee/tracee.go

```diff
@@ -713,6 +713,18 @@ func (t *Tracee) populateBPFMaps() error {
 	stringStoreMap, _ := t.bpfModule.GetMap("string_store")
 	stringStoreMap.Update(uint32(0), []byte("/dev/null"))
 
+	kernelConfig := helpers.KernelConfig{}
+	kconf, err := helpers.InitKernelConfig(kernelConfig)
+	if err != nil {
+		return fmt.Errorf("could not read in kernel configuration %v", err)
+	}
+
+	kernelConfigMap, _ := t.bpfModule.GetMap("kernel_config_map")
+	v, err := kconf.GetKernelConfigValue(helpers.CONFIG_BPF_SYSCALL)
+	if err == nil && v == "y" {
+		kernelConfigMap.Update(helpers.CONFIG_BPF_SYSCALL, uint32(1))
```

Comment:
Do we have a reason to populate CONFIG_BPF_SYSCALL? It should always be set in our case

Reply:
I had this somewhat for testing purposes, i'm going to change this to `KERNEL_CONFIG_DEBUG_INFO_BTF` as that's the most relevant feature (will be using this in follow up PR).